docker-compose.yml

```diff
@@ -31,6 +31,8 @@ services:
       RUST_BACKTRACE: 1
       SERVER_HOST: 0.0.0.0:8000
       RUST_LOG: info
+      OPENID_CLIENT_ID: ${OPENID_CLIENT_ID}
+      OPENID_CLIENT_SECRET: ${OPENID_CLIENT_SECRET}
```

Comment:
I think if you just don't pass a value, it'll automatically forward whatever value is in the env.

Reply:
like this?
```
OPENID_CLIENT_ID: 
OPENID_CLIENT_SECRET: 
```

It got set to null if i did that

Reply:
wait nvm, my `.env` file got wiped when i switched branches lol